h5c/vic/src/vic-webapp/src/app/create-vch-wizard/create-vch-wizard.component.html

```diff
@@ -130,7 +130,10 @@
     </ng-template>
     <ng-template clrPageNavTitle>Security</ng-template>
 
-    <vic-vch-creation-security [vchName]="generalStep.form.get('name').value" #securityStep></vic-vch-creation-security>
+    <vic-vch-creation-security [vchName]="generalStep.form.get('name').value"
+                               [datacenter]="computeCapacityStep.dcName"
```

Comment:
Is this right? I was told we couldn't (or shouldn't) use name but should use the MOID of the datacenter

Reply:
It worked with the name and not the MOID, I got an error form the CLI with the MOID and event suggest me to use that name.

Although the API needs the MOID. It fails with dc name.